packages/grafana-llm-app/src/components/AppConfig/OpenAI.tsx

```diff
@@ -45,22 +45,35 @@ export function OpenAIConfig({
         event.currentTarget.type === 'checkbox' ? event.currentTarget.checked : event.currentTarget.value.trim(),
     });
   };
+
+  // Update settings when provider changes, set default URL for OpenAI
+  const onChangeProvider = (value: OpenAIProvider) => {
+    onChange({
+      ...settings,
+      provider: value,
+      url: value === 'openai' ? 'https://api.openai.com' : '',
```

Comment:
nit: might be worth extracting this URL into a constant since we use it in a few places